### PR TITLE
Feature/add date to database

### PR DESCRIPTION
**Pull request by rosealti**
Extract date from xml

### CODE REVIEW

File: parser/crawler_current_events.py [hunk 1]
```diff
@@ -1,15 +1,17 @@
-"""module docstring"""
+"""module crawler"""
 import re
 import requests
 
 
 def grab_text(url):
+    """function docstring"""
```
Comment (datdo) [inline parser/crawler_current_events.py:7]: better doc string

File: parser/database_connection.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
+from sqlalchemy import create_engine
+import os
+
+
+def connect_to_database():
+    db = create_engine(os.environ.get('DB_CONNECTION_STRING'))
+    return db
+
+
+def create_and_insert_to_database(db):
```
Comment (datdo) [inline parser/database_connection.py:10]: what is the purpose of this?

File: parser/main.py [hunk 3]
```diff
@@ -6,13 +7,17 @@
 from primary_doc_xml import grab_primary_doc_root
```
Comment (datdo) [inline parser/main.py:7]: typically people use get instead of grab

File: parser/requirements.txt [hunk 4]
```diff
@@ -1,3 +1,5 @@
 requests
 sqlalchemy
-pandas
\ No newline at end of file
+pandas
+os
```
Comment (datdo) [inline parser/requirements.txt:4]: you don't need to put os. Its builtin

**Review by datdo — CHANGES_REQUESTED** [review]

File: parser/requirements.txt (hunk 4 above)
Comment (rosealti) [reply parser/requirements.txt:4]: This is old. See refactor/change_names.

File: parser/primary_doc_xml.py [hunk 5]
```diff
@@ -22,11 +22,21 @@ def grab_primary_doc_cik(primary_doc_root):
...
     namespaces = {'original': 'http://www.sec.gov/edgar/thirteenffiler',
                   'ns1': 'http://www.sec.gov/edgar/common'}
     for company_name in primary_doc_root.findall(
             'original:formData/original:coverPage/original:filingManager/original:name',
             namespaces):
         return company_name.text
+
+
+def get_primary_doc_accepted_filing_date(primary_doc_root):
+    """function docstring"""
```
Comment (datdo) [inline parser/primary_doc_xml.py:36]: docstring

**Review by datdo — CHANGES_REQUESTED** [review]

**Review by datdo — APPROVED** [review]

**Pull request merged**